### DISCUSSION TITLE
Cross browser execution for single test in data driven

### DISCUSSION THREAD

**Discussion by sseenivasan89** [testsigmahq/testsigma#38] [Feature Ideas]
Hi Team,

I want to execute the single test with different data using data driven
ex- I have 3 set of data and test cases need to run in different OS and browser

**Comment by diveshmehta06** [comment]
Hi @sseenivasan89 
Data Driven testing should not be browser specific, we recommend running test with all 3 data sets in all OS+browsers combinations
